src/storage/QueryBaseProcessor.inl

```diff
@@ -104,7 +124,18 @@ cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::checkAndBuildContexts(const REQ&
                 prop.retIndex_ = index++;
                 prop.prop_ = std::move(col);
                 prop.returned_ = true;
-                edgeContext_.props_.emplace_back(std::move(prop));
+                auto it2 = edgeContext_.find(edgeType);
+                if (it2 == edgeContext_.end()) {
+                    if (overAllEdge_) {
+                        std::vector<PropContext> v{std::move(prop)};
+                        edgeContext_.emplace(edgeType, std::move(v));
+                        break;
+                    } else {
+                        CHECK(it2 != edgeContext_.end());
```

Comment:
maybe is it2 == edgeContext_.end() ?

Reply:
When entering this condition, meaning we must update the it2->second, so the "edgeType" must exist in "edgeContext_".

Reply:
but this check is in the condition "it2 == edgeContext_.end()", so this check is always false.

Reply:
"it2 != edgeContext.end()" this  means edgeContext_ include edgeType.

Reply:
when overAllEdge_ is false, the logic as follow:
if (it2 == edgeContext_.end()) {
    CHECK(it2 != edgeContext_.end());
}

Reply:
yes, when overAllEdge_ was false, the edgeContext_ must include edgeType. so maybe i can using the CHECK(false).